### DISCUSSION TITLE
Cloudflare D1 vs Supabase?

### DISCUSSION THREAD

**Discussion by LouisDeconinck** [scosman/CMSaasStarter#121] [General]
Any reason in particular why you opted for Supabase as opposed to Cloudflare D1?

**Comment by scosman** [comment]
Just that Postgres is amazing, supabase offered hosted auth along side it, and supabase is open source/can self host.

**Reply by LouisDeconinck** [reply]
If I wanted to swap out Supabase for Cloudflare D1, how would I best go about that?

**Reply by scosman** [reply]
It would be a non trivial effort. But pull out the supabase dependency, delete SB code, and rebuild with D1.
